### PR TITLE
OCD-4437 - Create new report for listings that need to be updated for HTI-1

### PR DESCRIPTION
**Pull request by tmy1313**

### CODE REVIEW

File: chpl/chpl-resources/src/main/resources/system-triggers.xml [hunk 1]
```diff
@@ -5,6 +5,16 @@
...
     version="2.0">
     <schedule>
+        <trigger>
+            <cron>
+                <name>generateUpdateListingStatusReport</name>
+                <group>generateUpdateListingStatusReportTrigger</group>
+                <job-name>updatedListingStatusReportCreatorJob</job-name>
+                <job-group>systemJobs</job-group>
+                <misfire-instruction>MISFIRE_INSTRUCTION_DO_NOTHING</misfire-instruction>
+                <cron-expression>0 5 9 * * ?</cron-expression> <!-- At 0505 UTC every day -->
```
Comment (andlar) [inline chpl/chpl-resources/src/main/resources/system-triggers.xml]: The comment says 0505, but the cron job is, either 0509 or 0905. I can't recall which

File: chpl/chpl-resources/src/main/resources/email.properties [hunk 2]
```diff
@@ -193,6 +193,11 @@ listingValidationReport.heading=CHPL Listing Validation Report
 listingValidationReport.paragraph1.heading=Report contains data for the following ONC-ACBs
 listingValidationReport.paragraph2.body=%s validation errors found.
 
+#Updated Listing Status email properties
+updatedListingStatusReport.fileName=updated-listing_status_report
```
Comment (kekey1) [inline chpl/chpl-resources/src/main/resources/email.properties]: Not a big deal, but can we have all dashes or all underscores?

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/standard/CertificationResultStandard.java [hunk 3]
```diff
@@ -31,6 +32,9 @@ public class CertificationResultStandard implements Serializable {
     @JsonIgnore
     private Long certificationResultId;
 
+    @JsonIgnore
+    private Date createDate;
```
Comment (kekey1) [inline chpl/chpl-service/src/main/java/gov/healthit/chpl/standard/CertificationResultStandard.java]: I feel like everywhere else this is "creationDate". Any reason it's different or objection to changing it?

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/standard/CertificationResultStandard.java [hunk 4]
```diff
@@ -9,16 +10,16 @@
...
 import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
 
 @Schema(description = "The standards associated to the certification result")
 @JsonIgnoreProperties(ignoreUnknown = true)
 @NoArgsConstructor
 @AllArgsConstructor
 @Data
-@Builder
+@SuperBuilder
```
Comment (kekey1) [inline chpl/chpl-service/src/main/java/gov/healthit/chpl/standard/CertificationResultStandard.java]: Does something extend from this class now? Maybe I missed it in the PR.

**Review by kekey1 — DISMISSED** [review]

**Review by andlar — DISMISSED** [review]
Approving to go to QA, but will need to address comments before it's merged

**Review by kekey1 — DISMISSED** [review]

**Review by andlar — DISMISSED** [review]

**Review by andlar — DISMISSED** [review]

**Review by andlar — APPROVED** [review]

**Review by kekey1 — APPROVED** [review]

**Pull request merged**